Tests/YamsTests/EncoderTests.swift

```diff
@@ -94,8 +102,9 @@ import Yams
                 XCTFail("Failed to encode \(T.self) to YAML.")
             }
 
-            if let expectedYAML = yaml {
-                XCTAssertEqual(expectedYAML, payload, "Produced YAML not identical to expected YAML.")
+            if let expectedYAML = yamlString {
+                let producedYAML = String(data: payload, encoding: .utf8)! // swiftlint:disable:this force_unwrapping
```

Comment:
You can probably skip the force unwrap here as I _think_ that `XCTAssertEqual` can take arguments of `(String?, String)`